ifrs17-template/Report/InteractiveReports/Accruals.ipynb

```diff
@@ -0,0 +1,75 @@
+{
+    "metadata": {
+        "authors": [],
+        "id": "oKYWmrRFfU2e9YTd1Vj9iw",
+        "kernelspec": {
+            "display_name": "Formula Framework",
+            "language": "C#",
+            "name": "C#"
+        },
+        "language_info": {
+            "file_extension": ".cs",
+            "mimetype": "text/plain",
+            "name": "C#"
+        }
+    },
+    "nbformat": 4,
+    "nbformat_minor": 5,
+    "cells": [
+        {
+            "cell_type": "markdown",
+            "source": [
+                "<p style=\"font-weight:bold;\"> <span style=\"font-size: 36px\"> Accruals</span> </p>"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "#!import \"LoadData\""
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "Workspace.InitializeFrom(DataSource);",
```

Comment:
Why is this also not part of LoadData?

Reply:
I will add this initialization to LoadData and remove it here and all other NBs.

Reply:
done